hopper/src/main/java/org/atomhopper/abdera/parser/CategoryUtils.java

```diff
@@ -0,0 +1,68 @@
+
+package org.atomhopper.abdera.parser;
+
+import org.apache.abdera.Abdera;
+import org.apache.abdera.factory.Factory;
+import org.apache.abdera.model.Category;
+
+import java.util.*;
```

Comment:
Adding note for later: make imports explicit instead of wildcards when possible.